compute/admin_guide/waas/waas_api_discovery.adoc

```diff
@@ -43,12 +43,17 @@ Select a path to get more statistics on an endpoint and view the JSON structure.
 
 image::waas-api-discovery.png[scale=20]
 
-Select an endpoint to view the sidecar with additional statistics, such as request and response size ranges, and the sum of each specific returned status code.
+Select an endpoint to view the sidecar with additional statistics, such as request and response size ranges, the sum of each specific returned status code, and the API change history.
+
 The statistics also show the `json` payload that was sent with the API request.
 
 image::waas-api-discovery-sidecar1.png[scale=15]
 
-image::waas-api-discovery-sidecar2.png[scale=10]
+//image::waas-api-discovery-sidecar2.png[scale=10] The JSON payload already shown in the above screenshot
+
+The discovered API *Change History* log records the changes in the message structure, or authentication, or any new sensitive data match, but doens't detect the changes in the body of the API messages.
```

Comment:
@divyabhushan - changes to the message structure are not logged - please remove that.
Here's my new suggestion:

> API Change History logs detect changes in API observations such as authentication, query parameters, path risks or any new sensitive data matches, but not changes to the body of API messages.

Reply:
@e-shuster Thanks. I included this change in a new PR#923 [here](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/923).